rest/blip_api_test.go

```diff
@@ -1005,7 +1005,7 @@ function(doc, oldDoc) {
 }
 
 // Start subChanges w/ continuous=true, batchsize=20
-// Start goroutine sending rev messages for documents that grant access to themselves for the active replication's user
```

Comment:
The change to make the test single threaded seems to go against the whole point of the test (issuing concurrent grants to detect the race fixed by CBG-628) and I'm not sure solves the underlying timeout problem.

Is this the view performance issue you're currently investigating?

Reply:
I don't see how this change goes against the point of the test as a Goroutine is not being started for every `SendRev` and instead only wraps the loop that creates the revs in a goroutine. All this change does is mean that the timeout to receive the changes gets started once all the revs have been sent (as opposed to at the same time the loop starts to send the revs).